pootle/apps/pootle_translationproject/views.py

```diff
@@ -240,15 +239,19 @@ def get_context_data(self, *args, **kwargs):
             {'parent': get_parent(self.object)})
         return ctx
 
-    def get_upload_widget(self):
-        ctx = {}
-        has_upload = (
+    @property
+    def can_upload(self):
+        return (
             "import_export" in settings.INSTALLED_APPS
             and self.request.user.is_authenticated()
-            and check_permission('translate', self.request))
```

Comment:
hmm fair point but this method is full of implict wierdness - ill change the comit message